hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeRequest.java

```diff
@@ -95,13 +100,18 @@ protected VolumeList addVolumeToOwnerList(VolumeList volumeList,
     }
 
     List<String> prevVolList = new ArrayList<>();
+    long objectID = 100;
```

Comment:
Is there a reason for initialize objectID to 100 here for new volume?

Reply:
No, thanks I will fix that, I discovered it from a test code path, and left over from the fix. I will get this fixed.